src/main/kotlin/com/openlattice/chronicle/services/studies/StudyService.kt

```diff
@@ -647,6 +653,18 @@ class StudyService(
         }
     }
 
+    override fun getStudyFeatures(studyId: UUID): Map<StudyFeature, Map<String, Any>> {
+        return storageResolver.getPlatformStorage().connection.use { connection ->
```

Comment:
This should be read from the study cache.

Reply:
Reading from cache seems to return an empty map even though it's not empty
```
chronicle=# select modules from studies;
                                   modules                                    
------------------------------------------------------------------------------
 {"IOS_SENSOR": {}, "CHRONICLE_SURVEYS": {}, "CHRONICLE_DATA_COLLECTION": {}}

```
<img width="566" alt="Screen Shot 2022-04-08 at 10 00 27 AM" src="https://user-images.githubusercontent.com/15723547/162488181-e80d4712-560d-4e85-9cd1-140737383628.png">